[2023-04-06T13:32:06.330+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_dag_with_python_operator_v06.greet scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2023-04-06T13:32:06.376+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_dag_with_python_operator_v06.greet scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2023-04-06T13:32:06.376+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T13:32:06.376+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 6
[2023-04-06T13:32:06.376+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T13:32:06.526+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): greet> on 2022-05-22 00:00:00+00:00
[2023-04-06T13:32:06.556+0000] {local_task_job.py:113} ERROR - Received SIGTERM. Terminating subprocesses
[2023-04-06T13:32:06.565+0000] {local_task_job.py:212} INFO - Task exited with return code 143
[2023-04-06T13:32:06.574+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'create_dag_with_python_operator_v06', 'greet', 'scheduled__2022-05-22T00:00:00+00:00', '--job-id', '4646', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpirzxy3no']
[2023-04-06T13:32:06.592+0000] {standard_task_runner.py:83} INFO - Job 4646: Subtask greet
[2023-04-06T13:32:06.667+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-04-06T13:32:06.674+0000] {standard_task_runner.py:55} INFO - Started process 4691 to run task
[2023-04-06T13:32:06.679+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 4691. PIDs of all processes in the group: [4691]
[2023-04-06T13:32:06.679+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 4691
[2023-04-06T13:32:06.697+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=4691, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='13:32:05') (4691) terminated with exit code Negsignal.SIGTERM
